website/content/docs/integrations/vault/index.mdx

```diff
@@ -0,0 +1,60 @@
+---
+layout: docs
+page_title: Vault Integration
+description: |-
+ How to integrate Vault and Boundary
+---
+
+# Boundary and Vault overview
+The integration between Boundary and Vault aims to address two main areas of concern for organisations:
+
+- How to improve security posture in relation to remote access.
+- How to improve workflow efficiency.
+
+Boundary and Vault achieve these two things by removing the requirement for the end-user to have to know about what the credentials are or have anything to do with using them to gain access to a resource.
+
+From a security perspective, we can use dynamic, ephemeral credentials that are only valid for the lifetime of the session, unless a specific TTL has been attached to the Vault token that would result in a session having a finite amount of time.
+
+The security benefits also extend past an organisation's internal team, but also caters for third-party or contractors that may need to access certain resources. It would be preferable to not expose credentials for resource access with temporary staff, whilst still ensuring that access is secure and granted in a timely manner.
+
+Ensuring access is granted in a timely manner then leads to that improvement in workflow efficiency. This is the result of end-users not having to be concerned with credentials, therefore removing a large bottleneck, in relation to gaining credentials through access requests and the time associated with such a task.
+
+# Credentials
+Vault can work with Boundary to be a credential [store](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/credential-stores) and [library](https://developer.hashicorp.com/boundary/docs/concepts/domain-model/credential-libraries), which allows for credentials to be stored in Vault and used by Boundary. There are two options here for configuration:
+
+- Generic Secrets.
+- SSH Certificates.
+
+The Generic Secrets references a Vault Key-Value path, where static secrets are stored and can be used to broker credentials to the end-user when connecting to a target.
```

Comment:
Just re-reading this bit Dan. Could we maybe have something like:

"Generic secrets reference a Vault key-value path where static secrets are stored, e.g. username and password. You can use generic secrets to broker credentials to the end-user when they connect to a target."

I just want to highlight that this is an example and then it leads nicely into brokered and referencing the dynamic element for db credentials with Vault

Reply:
Yeah, I like that addition! I would just change "e.g." to "for example", because it will make more sense to an international audience. We try to avoid Latin phrases.

Reply:
Cool! I'll note that for the future. Aside from that I think it's a good start!